File: sc-petquote-angular/src/app/models/applicant.model.ts
export class ApplicantModel {
    public token = "";
    public leadId = "";
    public firstName = "";
    public lastName = "";
    public email = "";
    public contactNumber = "";
    public leadType = "pet qote";
    public channel = null;
    public leadScore = "";
    public status = "";
    public state = "";
    public quoteId = "";
    public isExistingPetCustomer = "";
    public isSCHSMember = "";
    public engagementChannel = "";
    public promoCode = "";
    public leadPreferences:Array<any> = new Array<any>();
    public pets:Array<ApplicantPetModel> = new Array<ApplicantPetModel>();
    public update(value: any):void {
        this.leadId = value.leadId;
        value.pets.forEach((value: any, index: number) => {
            this.pets[index].petId = value.petId;
        });
    }
}
export class ApplicantPetModel {
    public petId = "";
    public petName = "";
    public animal = "";
    public animalBreed = "";
    public dob = "";
    public gender = "";
}

File: PetInsurance-Site/Models/ApplicantModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PetInsurance_Site.Models
{
    public class ApplicantModel
    {
        public string token { get; set; }
        public string leadId  { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public string email { get; set; }
        public string contactNumber { get; set; }
        public string channel { get; set; }
        public string creationDate { get; set; }
        public string modifiedDate { get; set; }
        public string leadScore { get; set; }
        public string status { get; set; }
        public string state { get; set; }
        public string quoteId { get; set; }
        public List<ApplicantPetModel> pets { get; set; }

 
    }
    public class ApplicantPetModel
    {
        public string petId { get; set; }
        public string petName { get; set; }
        public string animal { get; set; }
        public string animalBreed { get; set; }
        public string dob { get; set; }
        public string gender { get; set; }
    }
}